### PR TITLE
chore: 🍰 Provide Volume For Backend Log-Files In Docker

### LINKED ISSUE #1904: 💥 [DevOps] Provide Volume For Backend Log-Files In Docker
## 💥 DevOps ticket

When running the backend in docker, now volume is bound to `../logs/`. This means that there are no log files on the disc. As long as we deploy without docker, this is no issue, but it would be nice to have the log files written with docker as well. 

### Additional Informations

see https://stackoverflow.com/questions/54374724/docker-container-save-logs-on-the-host-directory

### PR DESCRIPTION
**Pull request by Tirokk**
Provide volume for backend log-files in Docker.

### Issues

- fixes #1904

### Additional Informations

see https://stackoverflow.com/questions/42248198/how-to-mount-a-single-file-in-a-volume

### Todo

- [x] move provision of volume for ngnix from dev Docker Compose file to main
- [x] are the important log-files which I couldn't find as well for:
  - [x] database
    - is already in `database/typeorm.backend.log`
  - [x] frontend
    - nothing
  - [x] admin
    - nothing
  - [x] mariadb
    - [x] maybe: `/var/log`? I added this
      - but I can't see any logings
      - @mogge says we don't need this

### CODE REVIEW

**Comment by Mogge** [comment]
We have a `logs` folder, which is ignored by git. Inside this folder, there is a `backend` folder, which has four files:
```
access.log
apollo.log
backend.log
errors.log
```
When you run the seeds or unit tests locally, these files are appended. When you run the backend locally as well. When you run the backend in docker these files are not touched.

**Comment by Mogge** [comment]
The database log is in `database/typeorm.backend.log`. This works fine with docker.
Admin and Frontend have no Logs.

**Comment by Tirokk** [comment]
> `typeorm.backend.log`.

Then I mirror this file as well into the logs … @Mogge

**Comment by Tirokk** [comment]
https://github.com/gradido/gradido/pull/2067#issuecomment-1191276837
I understood this. @Mogge 

But with the `database/typeorm.backend.log` file it works from Docker somehow it looks like …

I pushed the new code …

**Review by Mogge — DISMISSED** [review]

File: docker-compose.yml [hunk 1]
```diff
@@ -133,6 +137,10 @@ services:
       # Application only envs
     #env_file:
     #  - ./frontend/.env
+    volumes:
+      # <host_machine_directy>:<container_directory> – mirror bidirectional path in local context with path in Docker container
+      # !!! this file has to exist !!! otherwise a folder is created at this path and we get a type error, because it's not a file
+      - ./logs/database/typeorm.backend.log:/app/typeorm.backend.log
```
[the remark this answers is not in the document]
Comment (Tirokk) [reply docker-compose.yml]: Done

**Review by Mogge — DISMISSED** [review]

**Comment by Tirokk** [comment]
!!! Please test crutial before you merge. @Mogge 

It has worked in the morning for the backend and works still for nginx, but now it does not work for the backend at the moment and I can not figure out why.

I restarted everything …

**Comment by Tirokk** [comment]
Now it should work as expected … @Mogge

**Review by Mogge — APPROVED** [review]
Now it is working! Great job!

**Pull request merged**